README.md

```diff
@@ -1,12 +1,56 @@
 veraPDF support for the PDF Arlington model
 =============================
 
-The [PDF Arlington model](https://github.com/pdf-association/arlington-pdf-model) covers the requirements of PDF object model as specified in PDF 2.0 and earlier. It is based on its own [formal grammar](https://github.com/pdf-association/arlington-pdf-model/blob/master/INTERNAL_GRAMMAR.md) serialized as a set of TSV files. 
+The [PDF Arlington Model](https://github.com/pdf-association/arlington-pdf-model) covers the requirements of PDF object model as specified in ISO 32000-2:2020 (PDF 2.0, including [resolved errata](https://pdf-issues.pdfa.org/)) as well as _some_ (but not all!) aspects from earlier Adobe PDF references and various extensions (identified by the predicate `fn:Extension(...)`). It is based on its own [formal grammar](https://github.com/pdf-association/arlington-pdf-model/blob/master/INTERNAL_GRAMMAR.md) serialized as a set of TSV files (see also the [Arlington Model Notes](https://github.com/pdf-association/arlington-pdf-model/blob/master/MODEL_NOTES.md)). References to Tables and clauses below all refer to ISO 32000-2:2020.
 
 veraPDF adds the support for this model by translating TSV files into its own validation profile based on [veraPDF formal grammar](https://docs.verapdf.org/validation/rules/) for validation rules. 
 
 This repository implements the Java-based utility performing this translation.
 
+Alington types
+==============================
+The Arlington PDF model utilizes an [expanded set of types](https://github.com/pdf-association/arlington-pdf-model/blob/master/INTERNAL_GRAMMAR.md#column-2---type) in order to more precisely define PDF objects and data integrity relationships. veraPDF fully supports this expanded set and implements the necessary additional checks. These checks include:
+- all dictionary keys must be direct objects
```

Comment:
veraPDF doesn't check that `all dictionary keys must be direct objects`. If a dictionary key is indirect object, we will throw exception `invalid pdf dictionary`.

Reply:
I assume this gets reported so isn't this the same thing?
Or does processing terminate?

Reply:
Yes, processing terminates.
![image](https://github.com/veraPDF/veraPDF-arlington-tools/assets/36183885/c2278ae2-7a4b-4dbd-bc96-bc63938d44be)

Reply:
New file pushed.